src/Sylius/Component/Core/Uploader/PathGeneratorInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Sylius\Component\Core\Uploader;
+
+use Gaufrette\Filesystem;
```

Comment:
Should we couple this class to Gaufrette? We could write adapter, I don't know, perhaps it a overkill.